integration/deploy/manifest_test.go

```diff
@@ -675,7 +674,7 @@ func createOktetoManifest(dir, content string) error {
 }
 
 func expectImageFoundSkippingBuild(output string) error {
-	if ok := strings.Contains(output, "Skipping build for image for service"); !ok {
+	if ok := strings.Contains(output, "Skipping build for image for service"); ok {
 		log.Print(output)
 		return errors.New("expected image found, skipping build")
```

Comment:
this will now throw an error if the skip is done, should we rename the condition? expectect no skipping?